src/commands/utility/vote-points.js

```diff
@@ -7,28 +7,45 @@ module.exports = {
   data: new SlashCommandBuilder()
     .setName('vote-points')
     .setDescription(
-      'Start a Effort Estimation Points votation for current Thread'
+      'Start an Effort Estimation Points votation for the current Thread'
     )
     .addUserOption((option) =>
       option
         .setName('user')
         .setDescription('User to search for')
         .setRequired(true)
+    )
+    .addStringOption((option) =>
+      option
+        .setName('point-type')
+        .setDescription('Select the type of points to award')
+        .setRequired(true)
+        .addChoices(
+          { name: 'Normal Points', value: 'normal' },
+          { name: 'Boosted Points', value: 'boosted' }
+        )
     ),
   async execute(interaction) {
     const user = interaction.options.getUser('user') || interaction.user;
+    const pointType = interaction.options.getString('point-type');
 
-    await interaction.reply(`<@&${tagIds.taskCompletedTagId}> <@${user.id}>`);
+    const selectedTagId =
+      pointType === 'boosted' ? tagIds.boostedPointTagId : tagIds.addPointTagId;
 
+    await interaction.reply(`<@&${tagIds.taskCompletedTagId}> <@${user.id}>`);
     interaction.followUp({
```

Comment:
this is probably async same that the .reply method
```suggestion
    await interaction.followUp({
```